Apps/WebClient/src/NewClientApp/src/components/private/timeline/entry/BcCancerScreeningResultTimelineComponent.vue

```diff
@@ -0,0 +1,110 @@
+<script setup lang="ts">
+import saveAs from "file-saver";
+import { computed, ref } from "vue";
+
+import HgButtonComponent from "@/components/common/HgButtonComponent.vue";
+import MessageModalComponent from "@/components/common/MessageModalComponent.vue";
+import TimelineEntryComponent from "@/components/private/timeline/TimelineEntryComponent.vue";
+import { EntryType, entryTypeMap } from "@/constants/entryType";
+import { container } from "@/ioc/container";
+import { SERVICE_IDENTIFIER } from "@/ioc/identifier";
+import { PatientDataFile } from "@/models/patientDataResponse";
+import BcCancerScreeningResultTimelineEntry from "@/models/timeline/bcCancerScreeningResultTimelineEntry";
+import { ILogger } from "@/services/interfaces";
+import { usePatientDataStore } from "@/stores/patientData";
+import SnowPlow from "@/utility/snowPlow";
+
+interface Props {
+    hdid: string;
+    entry: BcCancerScreeningResultTimelineEntry;
+    index: number;
+    datekey: string;
+    isMobileDetails?: boolean;
+    commentsAreEnabled?: boolean;
+}
+const props = withDefaults(defineProps<Props>(), {
+    isMobileDetails: false,
+    commentsAreEnabled: false,
+});
+
+const logger = container.get<ILogger>(SERVICE_IDENTIFIER.Logger);
+const patientDataStore = usePatientDataStore();
+const messageModal = ref<InstanceType<typeof MessageModalComponent>>();
+
+const entryIcon = computed(
+    () => entryTypeMap.get(EntryType.BcCancerScreening)?.icon
+);
+const isLoadingFile = computed(
+    () =>
+        props.entry.fileId !== undefined &&
+        patientDataStore.isPatientDataFileLoading(props.entry.fileId)
+);
+const hasFile = computed(
+    () => props.entry.fileId !== undefined && props.entry.fileId !== ""
+);
+
+function showConfirmationModal(): void {
+    messageModal.value?.showModal();
+}
+
+function downloadFile(): void {
+    if (props.entry.fileId) {
+        SnowPlow.trackEvent({
+            action: "download_report",
+            text: "BC Cancer Result PDF",
+        });
```

Comment:
```suggestion
        SnowPlow.trackEvent({
            action: "bc_cancer_result",
            text: "Download Report",
        });
```

Reply:
Ah this doesn't conform with the other downloads. We have a new ticket to redefine the snowplow data conventions etc. for now I want this one to stick the current methodology:
![image](https://github.com/bcgov/healthgateway/assets/19548348/ac509e72-b26a-43df-b766-076f8a208b0d)

Reply:
My suggestion contained the values from the story requirements.

Reply:
I understand, @ninosamson has phrased snowplow requirements during clarification as "however the others work" so I went with conformity, knowing we're going to visit this properly down the line.

Reply:
Just checking the AC - it is as below, which i believe aligns with how we do other data sets. For the recall story, I will be wanting a different "text" value of "bc_cancer_recall" 

action: Download Report
text: bc_cancer_result